### ISSUE TITLE
Last MM release (4.17.2) causes footer to always be displayed in full on our site

### ISSUE THREAD

**Issue by swcurran** [mmistakes/minimal-mistakes#2306]
## Environment

- Minimal Mistakes version: 4.17.2
- Ruby gem or remote theme version: gh-pages
- Jekyll version: gh-pages
- Git repository URL: https://github.com/bcgov/von
- GitHub Pages hosted (if yes provide URL to site): https://vonx.io
- Operating system: gh-pages (verified locally as well - linux)

## Expected behavior

Footer displays only when user scrolls to the bottom of the page. Footer is beneath the content.

Actual Behavior: Full footer is always showing from the bottom of the browser, so it is displaying over top of our content.

We fixed the problem by reverting back to the previous version of the theme - 4.17.0.

## Steps to reproduce the behavior

Display header page.  The bad behavior is in the image below.  The expected behavior is on the current version of the site (https://vonx.io).

![MMistakesFooterBug-4 17 2](https://user-images.githubusercontent.com/1018050/68088225-9b01db00-fe11-11e9-898d-557a9043fc93.jpeg)

## Other

The only change we made was to add the version to this line of the `_config.yml` file:

`remote_theme             : "mmistakes/minimal-mistakes@4.17.0"`

**Comment by mmistakes** [comment]
In 4.17.0 the theme was using JavaScript to add some whitespace to the body element to clear the sticky footer.

I removed that and went with flexbox solution. Your issue is you've customized the theme by overriding layouts, includes, etc and they are negating the new changes. You'll either need to remove these modifications or merge in the changes I made in this commit:

https://github.com/mmistakes/minimal-mistakes/commit/6d28d12d1dcf4883d41e168882c06f1d9948c040

Closing as it is a duplicate of #2301
